apps/fz_http/lib/fz_http/api_tokens/api_token.ex

```diff
@@ -0,0 +1,51 @@
+defmodule FzHttp.ApiTokens.ApiToken do
+  @moduledoc """
+  Stores API Token metadata to check for revocation.
+  """
```

Comment:
I removed Credo check for moduledocs, when they obvious it's better not to have them rather than having something very generic that just takes scroll space.